backend/fms_core/migrations/0028_v3_5_0.py

```diff
@@ -0,0 +1,96 @@
+from django.contrib.auth.models import User
+import reversion
+
+ADMIN_USERNAME = 'biobankadmin'
+
+
+def create_sample_qc_objects(apps, schema_editor):
+    Platform = apps.get_model("fms_core", "Platform")
+    InstrumentType = apps.get_model("fms_core", "InstrumentType")
+    Protocol = apps.get_model("fms_core", "Protocol")
+    PropertyType = apps.get_model("fms_core", "PropertyType")
+    ContentType = apps.get_model('contenttypes', 'ContentType')
+
+    with reversion.create_revision(manage_manually=True):
+        admin_user = User.objects.get(username=ADMIN_USERNAME)
+        admin_user_id = admin_user.id
+
+        reversion.set_comment("Create objects related to protocol Sample Quality Control")
+        reversion.set_user(admin_user)
+
+        # Create Platform and InstrumentType
+        p_quantitation = Platform.objects.create(name="Quality Control: Quantitation", created_by_id=admin_user_id, updated_by_id=admin_user_id)
+        p_electrophoresis = Platform.objects.create(name="Quality Control: Electrophoresis", created_by_id=admin_user_id, updated_by_id=admin_user_id)
+        reversion.add_to_revision(p_quantitation)
+        reversion.add_to_revision(p_electrophoresis)
+
+        # DICT { TYPE : PLATFORM }
+        INSTRUMENT_TYPES = {
+            "PicoGreen": "Quality Control: Quantitation",
+            "Qubit": "Quality Control: Quantitation",
+            "Agarose Gel": "Quality Control: Electrophoresis",
+            "TapeStation": "Quality Control: Electrophoresis",
+        }
+        for type in INSTRUMENT_TYPES:
+            platform = Platform.objects.get(name=INSTRUMENT_TYPES[type])
+            it = InstrumentType.objects.create(platform=platform,
+                                               type=type,
+                                               created_by_id=admin_user_id,
+                                               updated_by_id=admin_user_id)
+
+            reversion.add_to_revision(it)
+
+        # Create PropertyType and Protocols
+        # TODO: Maybe the dictionary is not necessary since there's only 1 protocol
+        PROPERTY_TYPES_BY_PROTOCOL = {
+            "Sample Quality Control": [("RIN", "str"),
+                                       ("Quantitation Instrument", "str"),
+                                       ("Electrophoresis Instrument", "str"),
+                                       ("Comment", "str")
+                                       ],
+        }
+        protocol_content_type = ContentType.objects.get_for_model(Protocol)
+
+        for protocol_name in PROPERTY_TYPES_BY_PROTOCOL.keys():
+            protocol = Protocol.objects.create(name=protocol_name,
+                                               created_by_id=admin_user_id, updated_by_id=admin_user_id)
+            reversion.add_to_revision(protocol)
+
+            for (property, value_type) in PROPERTY_TYPES_BY_PROTOCOL[protocol_name]:
+                is_optional = False if 'instrument' in property.lower() else True
```

Comment:
do not set stuff using string match ... it is much safer and clearer to set them directly. Add an if with the specific name of both instrument types and set them to false directly.